### PR TITLE
[4.0] Frontend module form button spacing

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue #29139.

### Summary of Changes

Fixes frontend module form button styling.

### Testing Instructions

Edit a module in frontend.

### Expected result

Buttons have some spacing:

Screenshot_2020-05-19 Home

### Actual result

Buttons have no spacing:

Screenshot_2020-05-19 Home(1)

### Documentation Changes Required

No.

### CODE REVIEW

File: components/com_config/tmpl/modules/default.php [hunk 1]
```diff
@@ -46,25 +46,19 @@
...
-				<div class="btn-group">
-					<button type="button" class="btn btn-danger" data-submit-task="modules.cancel">
-						<span class="fas fa-times" aria-hidden="true"></span>
-						<?php echo Text::_('JCANCEL') ?>
-					</button>
-				</div>
+			<div role="toolbar" aria-label="<?php echo Text::_('JTOOLBAR'); ?>">
+				<button type="button" class="btn btn-primary" data-submit-task="modules.apply">
+					<span class="fas fa-check" aria-hidden="true"></span>
+					<?php echo Text::_('JAPPLY') ?>
```
Comment (Quy) [inline components/com_config/tmpl/modules/default.php]: Add `;` for consistency.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on dfe45c82585246271d2d4c183a72be2e65fd8f71<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29143">issues.joomla.org/tracker/joomla-cms/29143</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on dfe45c82585246271d2d4c183a72be2e65fd8f71<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29143">issues.joomla.org/tracker/joomla-cms/29143</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29143">issues.joomla.org/tracker/joomla-cms/29143</a>.</sub>

**Comment by zero-24** [comment]
Merging thanks.

**Pull request merged**